style.css

```diff
@@ -27,8 +28,29 @@ body {
   margin: 0;
   padding: 0;
   list-style-type: none;
+ }
+
+.topic li 	{
```

Comment:
There is nothing located on the page with this selector. This is looking for a tag with the class `topic` and inside that tag an `li`.

Other ways to target the list items (one of which you discovered):

```css
li
.topic
li.topic
```